File: src/main/Java/boj_step8/boj2903.java
import java.io.*;

public class boj2903 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int n = Integer.parseInt(reader.readLine()) + 1, dots = 0, tmp = 2;
        while (n-- > 0) {
            dots = tmp * tmp;
            tmp *= 2;
            tmp--;
        }
        writer.append(String.valueOf(dots));
        writer.close();
    }
}

File: src/main/Kotlin/others/boj2903.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import java.time.temporal.ValueRange

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var n = r.readLine().toInt()
    n += 1
    var dots = 0
    var tmp = 2
    repeat(n) {
        dots = tmp * tmp
        tmp *= 2
        tmp -= 1
    }
    w.write(dots.toString())
    w.close()
}